documentation/src/main/wiki/ActionKnot.md

```diff
@@ -0,0 +1,35 @@
+ActionKnot is Knot.x system component that is responsible for handling forms submitting.
+
+Example:
+```html
+<script data-api-type="form-1" type="text/x-handlebars-template">
+  {{#if action._result.validationErrors}}
+  <p class="bg-danger">Email address does not exists</p>
+  {{/if}}
+  <p>Please provide your email address</p>
+  <form data-knotx-action="step1" data-knotx-on-success="/content/local/login/step2.html" data-knotx-on-error="_self" method="post">
+    <input type="email" name="email" value="{{#if action._result.validationError}} {{action._result.form.email}} {{/if}}" />
+    <input type="submit" value="Submit"/>
+  </form>
+  <div>
+    <strong>Pro tip: All emails that starts with <kbd>john.doe</kbd> will be accepted.</strong>
+  </div>
+</script>
+```
+
+In order to enable ActionKnot interpretation of a fragment, `data-api-type="form-{NAME}"` must be defined in `<script>` tag, 
```

Comment:
ActionKnot processes only those ''<script>' tags that have `data-api-type=form-{name}'